src/text/2-package.tex

```diff
@@ -1,6 +1,105 @@
+corresponding instrument response functions (IRFs). The latter include the aforementioned
+effective area, point spread function (PSF), energy dispersion and residual hadronic background.
+The handling of DL3 data is performed by classes and methods
+in the gammapy.data  (see \ref{ssec:gammapy-data}) and the gammapy.irf
+(see \ref{ssec:gammapy-irf}) subpackages.
+
+The first step in the analysis is the selection and extraction of observations
+based of their meta data including information such as pointing direction, observation
+time and observation conditions.
+
+The next step of the analysis is the data reduction on a user-defined geometry.
+A typical geometry consists in a spectral representation with a measured
+energy axis, and in a spatial representation, either a coordinates system  with a projection
+(for 3-dimensional or cube analysis) or a region on the sky (for regular spectral analysis).
+The gammapy.maps subpackage provides general multidimensional geometry objects
+(Geom) and the associated data structures (Maps), see \ref{ssec:gammapy-maps}.
+
+All observation events and instrument responses are projected onto the
+target geometry. Because residual hadronic background models can be subject
+to significant uncertainties background correction must be applied,
+such as the ring or the fied-of-view background techniques or
+background measurements must be performed within, e.g. reflected regions~\citep{Berge07}.
+Parts of the data with high associated IRF systematics must also be excluded by defining
+a "safe" data range. These data reduction steps are performed by classes and functions
+implemented in the gammapy.makers subpackage (see \ref{ssec:gammapy-makers}).
+
+The counts data and the reduced IRFs in the form of maps are bundled into dataset objects
+that represent the data level 4 (DL4). They can be written to
+disk, in a format specific to Gammapy to allow users to read them back at any time later
+for modeling and fitting.
+
+This latter step
+ datasets classes bundle reduced data in form of maps, reduced IRFs, models and
+fit statistics. Different sub-classes support different analysis methods
+and fit statistics (e.g. Poisson statistics with known background or
+with OFF background measurements). The datasets are used to perform joint-likelihood
+fitting allowing to combine different measurements, e.g. from different observations
+but also from different instruments or event classes. They can also be used for binned
+simulation as well as event sampling to simulate DL3 events data.
+
+The next step is then typically to model and fit the datasets, either
+individually, or in a joint likelihood analysis. For this purpose Gammapy
+provides a uniform interface to multiple fitting backends. It also provides
+a variety of :ref:`built in models <model-gallery>`. This includes spectral,
+spatial and temporal model classes to describe the gamma-ray emission in the sky.
```

Comment:
Suggestion:
Note that the spectral models can either be analytical (e.g. Power-laws) or from the radiation mechanisms of accelerated particle populations (e.g. inverse Compton or $\pi^{o}$ decay).